packages/reactivity/src/baseHandlers.ts

```diff
@@ -11,6 +11,10 @@ const builtInSymbols = new Set(
     .filter(isSymbol)
 )
 
+function isNaN(value: any, oldValue: any): boolean {
+  return value === value && oldValue === oldValue
+}
```

Comment:
```
value === value || oldValue === oldValue
```
One of them is `NaN` should be trigger, and the function name is confusing.

Reply:
You should do the same in `collectionHandlers.ts`, and add unit test for them.

Reply:
I think maybe ``collectionHandlers.ts`` is not to be judged

Reply:
Yes, it's my mistake.

Reply:
`collectionHandlers.ts` should to be judged when target is collection types(just like `Map/Set...` ), it the same when using `.set`.

Reply:
yes, but ``Set/WeakSet`` not have ``.set``

Reply:
NaN is equal in ``Set``，and ``WeakSet`` only add ``object``，I don't know I need to deal with it or not

Reply:
only `MapTypes`(`Map/WeakMap`) have `.set`, and it should be work. 
only `SetTypes`(`Set/WeakSet`) have `.add`, but not need to check value.

Reply:
yes, i think it doesn't matter for this judgment now

Reply:
why? it doesn't matter for `.add`, but it should judgment in `.set` because we need to support `MapTypes`.

Reply:
right, i already add this judgment in ``collectionHandlers.ts``，and add test

Reply:
yeah, you can `Reslove conversation`.